Corona-Warn-App/src/main/res/layout/fragment_risk_details.xml

```diff
@@ -269,11 +269,7 @@
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
                         android:layout_marginTop="@dimen/spacing_normal"
-                        android:text="@string/risk_details_explanation_faq_body_with_link"
                         android:visibility="@{FormatterRiskHelper.formatVisibilityBehaviorLowLevelRisk(tracingViewModel.riskLevel)}"
-                        android:focusable="true"
```

Comment:
Can this be safely removed? Did you check on older Android versions?

Reply:
I'll have to check on the older android version but have no device for that.

Best is to keep that parameter then.